site/content/docs/user-guide/setup/install.md

```diff
@@ -24,10 +24,10 @@ Krew self-hosts).
       set -x; cd "$(mktemp -d)" &&
       OS="$(uname | tr '[:upper:]' '[:lower:]')" &&
       ARCH="$(uname -m | sed -e 's/x86_64/amd64/' -e 's/\(arm\)\(64\)\?.*/\1\2/' -e 's/aarch64$/arm64/')" &&
-      curl -fsSLO "https://github.com/kubernetes-sigs/krew/releases/latest/download/krew.tar.gz" &&
-      tar zxvf krew.tar.gz &&
-      KREW=./krew-"${OS}_${ARCH}" &&
-      "$KREW" install krew
+      KREWNAME="krew-${OS}_${ARCH}" &&
```

Comment:
Why didn't you like `KREW`?
```suggestion
      KREW="krew-${OS}_${ARCH}" &&
```

Reply:
As commented further, bash and fish installs were using different variable names. I thought `KREWNAME` was more appropriate as the var is used for the name of the tarball and things, but `KREW` is OK for me.